### ISSUE TITLE
Elements in share modal not accessible

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4189]
**Describe the bug**
Elements in Share modal are not accessible via keyboard.

**Where was the bug observed**
views/map in v5 deployment

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to the /views/maps
2. Click on the share button in top right corner.
3. Observe that the social media platform are not accessible via keyboard.

**Screenshots**
![image](https://user-images.githubusercontent.com/70752431/102889514-cf56b180-4480-11eb-8904-878689ee888e.png)
